pynamodb_attributes/unicode_delimited_tuple.py

```diff
@@ -51,7 +51,7 @@ def deserialize(self, value: str) -> T:
     def serialize(self, value: T) -> str:
         if not isinstance(value, self.tuple_type):
             raise TypeError(f"value has invalid type '{type(value)}'; expected '{self.tuple_type}'")
-        values = [e for e in value]
+        values = list(value)
```

Comment:
unrelated change? (though I agree in spirit)

Reply:
Yes, but Travis gave me a red x for “ '[C416] Unnecessary list comprehension'”. Not sure if the linter added a rule recently or what

Reply:
Since `flake8-comprehensions` is unpinned this kind of thing may happen from time to time. We could pin it